include/robot_localization/ros_filter_utilities.hpp

```diff
@@ -82,7 +82,7 @@ double getYaw(const tf2::Quaternion quat);
 //! value of @p targetFrameTrans and returns true, otherwise it returns false.
 //!
 bool lookupTransformSafe(
-  const tf2_ros::Buffer & buffer,
+  const tf2_ros::Buffer * buffer,
```

Comment:
Passing smart pointers to functions is considered against the best practices for C++. Not to say I didn’t do that because of that reason, I just did what required the least amount of effort :-) 

With that said, the smart pointers best practices are still developing.